number guessing game/extension_functions.py

```diff
@@ -0,0 +1,23 @@
+"""
+Game extension functions. These functions add extra functionalities to the game.
```

Comment:
Add a module description that explains what this module is doing in not less than three lines.

Reply:
Thanks for pointing that out.